aspnet/whitepapers/aspnet-and-iis6.md

```diff
@@ -207,8 +207,8 @@ ASP.NET 1.0 on IIS 6.0 is only supported in IIS 5 compatibility mode.
 
 To configure ASP.NET 1.0 to run in IIS 5.0 compatibility mode, open Internet Services Manager and right click Web Sites and select properties:
 
-![](aspnet-and-iis6/_static/image27.jpg)
+![A screenshot of the Windows I I S Manager screen. The  menu has the folder titled Web Sites highlighted.](aspnet-and-iis6/_static/image27.jpg)
```

Comment:
A screenshot of the Windows I I S Manager screen. The web sites folder is highlighted.